target/linux/meson/files/arch/arm/boot/dts/meson8b-onecloud.dts

```diff
@@ -0,0 +1,389 @@
+// SPDX-License-Identifier: GPL-2.0 OR MIT
```

Comment:
Please submit this dts to upstream kernel, as we don't want to handle this forever